com.insynchq.Insync.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.insynchq.Insync
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base-version: "24.08"
+command: /app/usr/lib/insync/insync
+separate-locales: false
+rename-app-id: com.insynchq.Insync
+rename-desktop-file: insync.desktop
+rename-icon: insync
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+    # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.kde.StatusNotifierItem
+  - --own-name=org.kde.*
+  # Needed for /dev/disk/by-id which insync accesses
+  - --device=all
+  - --filesystem=~/Insync
+  - --persist=.config
+  - --filesystem=xdg-config/autostart:create
+build-options:
+  strip: true
```

Comment:
```suggestion
```

why?

Reply:
It is because some binaries do not work after being stripped.

    /usr/lib/insync/shiboken2/shiboken2.abi3.so
    /usr/lib/insync/libshiboken2.abi3.so.5.15

And they are completely not loadable. That is all the loadable sections get removed.